homeassistant/components/supla/__init__.py

```diff
@@ -39,25 +49,27 @@
 )
 
 
-def setup(hass, base_config):
+async def async_setup(hass, base_config):
     """Set up the Supla component."""
 
     server_confs = base_config[DOMAIN][CONF_SERVERS]
 
     hass.data[SUPLA_SERVERS] = {}
-    hass.data[SUPLA_CHANNELS] = {}
+    hass.data[SUPLA_COORDINATORS] = {}
 
```

Comment:
i feel like this should be part of hass.data[DOMAIN][DATA_COORDINATORS] but i guess that means thr servers should move as well

Reply:
Agreed.